src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
```

Comment:
Header comments are missing. Make sure NOT to mention the child classes in the header comments of a super class.